freqtrade/tests/test_freqtradebot.py

```diff
@@ -993,6 +993,44 @@ def test_check_handle_timedout_buy(default_conf, ticker, limit_buy_order_old, fe
     assert nb_trades == 0
 
 
+def test_check_handle_timedout_buy_exception(default_conf, ticker, limit_buy_order_old,
```

Comment:
is it deliberate or it is a typo?  "timedout"

Reply:
i am not sure it's a typo - what would be the correct word?
... it checks if it "timed out" (so historic tense) - at least that's my assumption as to why the original function is called this way